docs/pages/get-started/installation.md

```diff
@@ -28,26 +29,26 @@ To install and use Expo CLI, you need to have the following tools installed on y
 - [Yarn](https://classic.yarnpkg.com/en/docs/install)
 - Windows users: [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows), Bash via WSL, or the VS Code terminal
 
-### Installing Expo CLI
+### Using Expo CLI
 
-To install Expo CLI, you need to install it as a global npm package. Open the terminal on your development machine and run the following command:
+To use Expo CLI, you can use it with `npx` &mdash; a Node.js package runner. For example, to see a list of available commands in Expo CLI, open the terminal on your development machine and run the following command:
```

Comment:
```suggestion
You can use Expo CLI without installation by leveraging `npx` &mdash; a Node.js package runner. For example, to see a list of available commands in Expo CLI, open the terminal on your development machine and run the following command:
```

The main goal was to remove multiple `use`, but I ended up altering a little bit more. Feel free to change this sentence in other way, which you think suits better.